models/comment.js

```diff
@@ -14,17 +14,19 @@ async function createComment(planId, userId, text) {
     let results = await pool.query(sql, [planId, userId, text]);
     const commentId = results[0].insertId;
 
+    // get the time that the comment was created
     sql = "SELECT time FROM Comment WHERE commentId=?;";
     results = await pool.query(sql, [commentId]);
+    const time = results[0][0].time;
+
+    // send out notifications about the new comment
+    planNotification(planId, userId, 1);
```

Comment:
`await`?

Reply:
So I want to get your opinion here. 

If the getLiveCourses function has no await. Should the notifications it spawns have awaits?
I think logically they should only have awaits if we will perform some action if the notifications fail to send.

This raises the question, should we perform an action if they fail to send?
Should we cancel the update if the notification at the beginning or end of the update fails?